compositor_pipeline/src/pipeline/input/rtp.rs

```diff
@@ -83,6 +73,27 @@ impl RtpReceiver {
             },
         ))
     }
+
+    fn create_socket(port: u16) -> std::io::Result<UdpSocket> {
+        let socket = socket2::Socket::new(
+            socket2::Domain::IPV4,
+            socket2::Type::DGRAM,
+            Some(socket2::Protocol::UDP),
+        )?;
+
+        // This doesn't fail if the requested size is larger than the system limit
+        // JK it does on macOS :^)
+        // TODO: Handle this gracefully
```

Comment:
gracefully?  I would imagine we just want to ignore the error and maybe log a warn?

Reply:
this is a WIP commit and this line was written by copilot because I was in a hurry. I will do this properly tomorrow.